.github/actions/build/action.yml

```diff
@@ -0,0 +1,36 @@
+name: Build
+
+runs:
+  using: "composite"
+  steps:
+    - name: Configure rustup
+      id: rustup
+      run: |
+        rustup update
+        rustup toolchain install --profile minimal nightly
+        rustup target add --toolchain nightly x86_64-pc-windows-msvc
+      shell: bash
+
+    - uses: actions/cache@v2
+      with:
+        path: |
+          ~/.cargo/registry/index/
+          ~/.cargo/registry/cache/
+          autofix/
+          labeler/
+          src/
+          target/
+        key: build-${{ github.run_id }}
+        restore-keys: build-
+
+    # We restore source files from the cache to keep their modification times,
+    # then use git to undo that for files that have actually changed. This
+    # prevents unnecessary rebuilds
+    - run: |
+        git status
+        git checkout .
+        git clean -f .
+      shell: bash
```

Comment:
Btw syntax seems wrong.

Reply:
How so? I can't see it

Reply:
Oh well GitHub's highlighting seems wrong then:
![image](https://user-images.githubusercontent.com/25030997/149434239-644a0549-cc1a-4663-a9f3-96db72455d6e.png)
Just checked on my editor and there's no problem, weird...

Reply:
ooh I see it now! tricky to spot with the light theme

![image](https://user-images.githubusercontent.com/1830331/149621444-0f480443-b05d-4cb4-852d-6a4bcb6a5bd6.png)

Yeah, odd, guess it's a GH bug